x-pack/plugins/monitoring/server/lib/logs/detect_reason.ts

```diff
@@ -24,7 +24,14 @@ async function doesFilebeatIndexExist(
   const metric = { timestampField: '@timestamp' };
   const filter = [createTimeFilter({ start, end, metric })];
 
-  const typeFilter = { term: { 'service.type': 'elasticsearch' } };
+  const typeFilter = {
+    bool: {
+      should: [
+        { term: { 'service.type': 'elasticsearch' } },
+        { term: { 'data_stream.type': 'logs' } },
```

Comment:
The duplication between this and `getLogTypes` is a little concerning. What happens if they diverge by accident.

Reply:
fixed in https://github.com/elastic/kibana/pull/139121/commits/7ed3bbfbce5cffafa4e4bdbccaa58e9a7c868dea